### PR TITLE
Successful Defibs are more costly, chargewise.

### PR DESCRIPTION
**Pull request by Anewbe**
From 1000 to 1250, and a smaller cell, borg zappers are unaffected.

### CODE REVIEW

**Comment by mistyLuminescence** [comment]
But this is even worse than the last one, because we can't take out the cells and upgrade them with the relatively-common high-cap cells... again, pretty sure the defib problem is caused by antags never sticking around after a kill to do anything about it.

**Comment by Wickedtemp** [comment]
This would lead to entire defibs being used up on a single patient, depending on how slow they are with returning to their body after ghosting. Usually, you'd zap 3-4 times. This isn't _defib spam_, because it's kind of the only way to defib someone who's ghosted. The first or second defib lets them know someone's trying to bring them back, and during the second or third, they have a little window to return to their body. 

"Defib Spam" is when you defib someone, they die, you defib them again, they die, rinse and repeat. Which I agree, it should be discouraged. Feel free to tack on burn damage on patients who are _being brought back by defibs repeatedly, not to corpses who keep popping up with MIF's with each attempt_.

**Comment by mistyLuminescence** [comment]
If this nerfs _unsuccessful_ defibs (read: ones where the patient doesn't come back to life), the title's pretty misleading - I was under the impression this would only work on defibs that, well, actually work. Using three defib charges on one body in one go is normal for medical.

**Comment by Wickedtemp** [comment]
At least partially due to defibs not even messaging ghosted players that doctors are attempting to defib them, apparently. 

https://gyazo.com/99483222545f77c130751d42d88fc1bb

**Comment by mistyLuminescence** [comment]
^ #4283 

Consider:
1) defib zaps that don't rez someone use no charge
2) defib zaps that do rez someone use double charge
and 3) we fix 4283 already (working on it!)

**Comment by mistyLuminescence** [comment]
TL;DR version now: APC cell (5k), each charge uses 1250, 4 uses per cell.
I'd support this, even if unsuccessful defib zaps use charge - with 4283 fixed, we won't need to defib-spam MIF patients to get them to come back.

**Comment by Wickedtemp** [comment]
I can get behind these changes.
